src/middlewared/middlewared/etc_files/keepalived.conf.mako

```diff
@@ -0,0 +1,58 @@
+<%
+    licensed = middleware.call_sync('failover.licensed')
+    node = middleware.call_sync('failover.node')
+
+    if licensed and node == 'MANUAL':
+        middleware.logger.error(
+            'Chassis position could not be determined.'
+            ' Keepalived config not generated.'
+        )
+	raise FileShouldNotExist()
+
+    config = middleware.call_sync('failover.config')
+
+    advert_int = [.68] # default 2 second timeout
```

Comment:
Because I don't know how to make mako write a single variable by itself. This was the easiest solution at the time. If you know how, please enlighten me 😄

Reply:
nvm, figured it out 😉